mulesoft-vectors/1.0/antora.yml

```
@@ -0,0 +1,18 @@
+name: mulesoft-vectors-connector
+version: '1.0'
+display_version: 1.0 (Mule 4)
+title: MuleSoft Vectors Connector
+nav:
+- modules/ROOT/nav.adoc
+asciidoc:
+  attributes:
+    page-component-desc: Enables you to integrate AI into your data for smarter search and RAG
+    page-connector-type: Connector
+    page-connector-level: MuleSoft Certified
```

```suggestion
    page-connector-level: MuleSoft Select 
```
 the connector will be labeled as "Select", not certified